.github/CODEOWNERS

```diff
@@ -84,6 +84,7 @@ BUILD @christophermaier
 /src/rust/nomad-client-gen/ @wimax-grapl
 /src/rust/rust-proto/ @grapl-security/wg-data-infra
 /src/rust/rust-toolchain.toml @inickles-grapl
+/src/rust/schema-manager @colin-grapl @wimax-grapl
```

Comment:
# We may change the name of this service 
to like, graph-schema-manager or org-schema-manager
maybe registry?

discussion here https://grapl-internal.slack.com/archives/C03JSKXNM9U/p1660836387554339